nomad/structs/structs.go

```diff
@@ -302,6 +303,13 @@ type NodeUpdateDrainRequest struct {
 	WriteRequest
 }
 
+// NodeUpdateEligibilityRequest is used for updating the scheduling	eligibility
```

Comment:
Fell asleep on the spacebar for a second

Reply:
Hm must be a UI thing? In code it is one space